### ISSUE TITLE
Include default support Visual Studio 2012+ dark theme

### ISSUE THREAD

**Issue by sharwell** [tunnelvisionlabs/LangSvcV2#3]
The default installation should include color settings for both the light and dark Visual Studio themes.

**Comment by RxDave** [comment]
This would be really nice. I'm using the dark them in VS 2017 and the tokens in dark blue are unreadable. Thanks for your hard work on this project!

**Comment by ATofler-Quest** [comment]
+1

**Comment by biiiipy** [comment]
Yes, this drives me nuts, pleae add dark theme support! (who event uses VS light theme? 😄)
